internal/controller/api/v1alpha1/installenv_types.go

```diff
@@ -84,6 +100,7 @@ type InstallEnvStatus struct {
 	// ISODownloadURL specifies an HTTP/S URL that contains a discovery ISO containing the
 	// configuration from this InstallEnv.
 	ISODownloadURL string `json:"isoDownloadURL,omitempty"`
+	State          string `json:"state,omitempty"`
```

Comment:
Maybe it need to be a condtion? @avishayt what to you think?

Reply:
Yes definitely.  I guess one condition should be enough right?

Reply:
Sure